.gitignore

```diff
@@ -14,6 +14,7 @@ build/
 .rvmrc
 .rspec-local
 /coverage
+.idea/
```

Comment:
this should go in your global .gitignore, see instructions here https://help.github.com/articles/ignoring-files/#create-a-global-gitignore 🙂